docs/replication/group-replication-flow-control.md

```diff
@@ -0,0 +1,18 @@
+# Manage Group Replication flow control
+
+In replication, flow control prevents one member from falling too far behind the cluster and avoid excessive buffering. A cluster is not required to keep members in sync together for replication. The pending transactions in the relay log only increase for the lagging replica. 
```

Comment:
In group replication / for group replication

Reply:
updated.